src/contracts/core/AVSDirectory.sol

```diff
@@ -63,6 +63,22 @@ contract AVSDirectory is
      *                    EXTERNAL FUNCTIONS
      *
      */
+    
+    /// @inheritdoc IAVSDirectory
+    function claimIdentifier(bytes20 avs) external {
```

Comment:
why not bytes32?

Reply:
need to encode with id for enumerable set